tribits/ci_support/CreateIssueTrackerFromCDashQuery.py

```diff
@@ -0,0 +1,271 @@
+import os
+import sys
+import pprint
+
+import CDashQueryAnalyzeReport as CDQAR
+import cdash_build_testing_date as CDBTD
+from Python2and3 import distItems
+
+g_pp = pprint.PrettyPrinter(indent=2)
+
+
+# Class that is used to create issue trackers for nonpassing tests downloaded
+# from a CDash queryTests.php query.
+#
+# To make this general, the user has to create a class object that is passed
+# in as the argument 'issueTrackerFormatter' in this class that supports
+# the following functions.
+#
+# issueTrackerFormatter.acceptIssueTrackerData(issueTrackerData):
+#
+# This function is given data about the nonpasing tests through an object of
+# type 'IssueTrackerData' as the argument 'issueTrackerData'.  This data is
+# used to create the issue issue tracker text.
+# issueTrackerFormatter.getIssueTrackerText():
```

Comment:
You want a blank line before this one?

Reply:
Actually, this documentation is wrong.  I merged these two functions into one function and forgot to update the documentation.  I have this fixed in my local branch.